modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -286,6 +303,19 @@ void createExoplanetSystem(const std::string& starName,
             "Period = " + std::to_string(periodInSeconds) + ""
         "}";
 
+        std::string planetLayers = "";
+        if (!planetTexture.empty()) {
+            planetLayers = std::format(
+                "ColorLayers = {{ {} }}",
+                "{"
+                    "Identifier = 'PlanetTexture',"
+                    "FilePath = openspace.absPath('" + formatPathToLua(planetTexture) + "'),"
+                    "BlendMode = 'Color',"
+                    "Enabled = true"
+                "}"
+            );
+        }
```

Comment:
```suggestion
            planetLayers = std::format(
                "ColorLayers = {{"
                "  {{"
                "    Identifier = 'PlanetTexture',"
                "    FilePath = openspace.absPath('{}'),"
                "    BlendMode = 'Color',"
                "    Enabled = true"
                "  }}"
                "}}",
                formatPathToLua(planetTexture)
            );
        }
```

Might be easier to read?

Reply:
Heh, yeah I agree that this is a bit odd.  
I'd just remove the std::format entirely, I think. To be more consistent with the rest of the function :)